pfcli/service/client/metrics.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2022 FriendliAI
+
+"""PeriFlow MetricsClient Service"""
+
+import json
+from typing import List
+
+from pfcli.service.client.base import ClientService, safe_request
+
+
+class MetricsClientService(ClientService):
+    """PeriFlow MetricsClientService class"""
+
+    def list_metrics(
+        self,
+        job_id: int,
+    ) -> List[str]:
+        """Show available metrics."""
+        fields = f'task: "{job_id}", telemetry: null, start: null, until: null'
+        query = "{ sampleQuery(fields: {" + fields + "}) {name} }"
+        resp = safe_request(
+            self.post, err_prefix=f"Failed to get metrics of ({job_id})."
+        )(
+            headers={"Content-Type": "application/json"},
+            data=json.dumps({"query": query}),
+        )
+        data = resp.json()["data"]["sampleQuery"]
+        names = set(sample.get("name", None) for sample in data)
+        return [{"name": name} for name in names if name]
+
+    def get_metrics_values(
+        self,
+        job_id: int,
+        name: str,
+        limit: int = 10,
+    ) -> dict[str, str]:
```

Comment:
`List[Dict[str, Any]]`? If we have type constraint for metric value, we can define Type variable `MetricType`.